components/Map/Map.js

```diff
@@ -0,0 +1,32 @@
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
```

Comment:
you aren't using `Marker` and `Popup` in this component.

```suggestion
import { MapContainer, TileLayer } from "react-leaflet";
```

Reply:
Hi Niklas,
thank you for reviewing!